### ISSUE TITLE
The AppCenter banner is broken

### ISSUE THREAD

**Issue by rasmus91** [Lafydev/Animage#8]
Something seems to be wrong with some tags placed in the AppCenter banner text. Screenshot provided to show what i mean.

![image](https://user-images.githubusercontent.com/2297739/123795457-54b19780-d8e4-11eb-8703-69d9c565bc34.png)

**Comment by Lafydev** [comment]
Yes, thanks, you're right ! The description is taken from the file "appdata.xml". Hum yes, to be fixed.
